### PR TITLE
change alpha1 to alpha2 in AC and Eventing tutorials

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Change Function CRD version from alpha1 to alpha2 in Developer Tutorials

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/16375

### CODE REVIEW

File: docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md [hunk 1]
```diff
@@ -36,29 +36,40 @@ You learn how Eventing behaves when you create a [Subscription](../../05-technic
...
+       serverless.kyma-project.io/build-resources-preset: local-dev
+       serverless.kyma-project.io/function-resources-preset: S
+       serverless.kyma-project.io/replicas-preset: S
+     name: lastorder
+     namespace: default
+   spec:
+     replicas: 1
+     resourceConfiguration:
+       function:
+         profile: XS
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md:52]: IMO this is in conflict with label `serverless.kyma-project.io/function-resources-preset: S`

File: docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md [hunk 2]
```diff
@@ -36,29 +36,40 @@ You learn how Eventing behaves when you create a [Subscription](../../05-technic
...
-           main: async function (event, context) {
-             console.log("Received event: ", event.data, ", Event Type: ", event.extensions.request.headers['ce-type']);
-             return; 
-           } 
-         }
+   apiVersion: serverless.kyma-project.io/v1alpha2
+   kind: Function
+   metadata:
+     labels:
+       serverless.kyma-project.io/build-resources-preset: local-dev
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md:43]: Now we rather prefer these settings inside `resourceConfiguration` except `serverless.kyma-project.io/replicas-preset`

File: docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md [hunk 3]
```diff
@@ -36,29 +36,40 @@ You learn how Eventing behaves when you create a [Subscription](../../05-technic
...
+       serverless.kyma-project.io/function-resources-preset: S
+       serverless.kyma-project.io/replicas-preset: S
+     name: lastorder
+     namespace: default
+   spec:
+     replicas: 1
+     resourceConfiguration:
+       function:
+         profile: XS
+         resources:
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md:53]: Should we use `profile` and `resources` together?

File: docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md [hunk 4]
```diff
@@ -39,29 +39,40 @@ The "in-flight messages" config defines the number of events that Kyma Eventing
...
-       minReplicas: 1
-       source: |
-         module.exports = {
-           main: async function (event, context) {
-             console.log("Processing event:", event.data);
+   apiVersion: serverless.kyma-project.io/v1alpha2
+   kind: Function
+   metadata:
+     labels:
+       serverless.kyma-project.io/build-resources-preset: local-dev
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md:46]: Now we rather prefer these settings inside `resourceConfiguration` except `serverless.kyma-project.io/replicas-preset`. In general I have the same notices like for `docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md`

File: docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md [hunk 5]
```diff
@@ -36,29 +36,40 @@ You learn how Eventing behaves when you create a [Subscription](../../05-technic
...
+           limits:
+             cpu: 100m
+             memory: 128Mi
+           requests:
+             cpu: 50m
+             memory: 64Mi
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md:59]:
```suggestion
     name: lastorder
     namespace: default
   spec:
     replicas: 1
     resourceConfiguration:
       function:
         profile: S
       build:
         profile: local-dev
```

File: docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md [hunk 6]
```diff
@@ -39,29 +39,40 @@ The "in-flight messages" config defines the number of events that Kyma Eventing
...
+     resourceConfiguration:
+       function:
+         profile: XS
+         resources:
+           limits:
+             cpu: 100m
+             memory: 128Mi
+           requests:
+             cpu: 50m
+             memory: 64Mi
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md:62]:
```suggestion
     name: lastorder
     namespace: default
   spec:
     replicas: 1
     resourceConfiguration:
       function:
         profile: S
       build:
         profile: local-dev
```

File: docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md [hunk 7]
```diff
@@ -39,29 +39,40 @@ The "in-flight messages" config defines the number of events that Kyma Eventing
...
+     runtime: nodejs16
+     source:
+       inline:
+         source: |-
+           module.exports = {
+             main: async function (event, context) {
+               console.log("Processing event:", event.data);
              // sleep/wait for 5 seconds
              await new Promise(r => setTimeout(r, 5 * 1000));
              console.log("Completely processed event:", event.data);
```
Comment (anoipm) [inline docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md:72]:
```suggestion
               // sleep/wait for 5 seconds
               await new Promise(r => setTimeout(r, 5 * 1000));
               console.log("Completely processed event:", event.data);
```

**Review by anoipm — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**